.travis.yml

```diff
@@ -25,7 +28,19 @@ cache:
   - .gradle
   - $HOME/.gradle
   - $HOME/.m2
-  
+
+stages:
+  - name: build
+  - name: snapshot
+    if: branch = issue-119
```

Comment:
I will change this to `master` before merging the code. It is pointing to the branch for testing purpose.